core-api/src/entities/place.entity.ts

```diff
@@ -0,0 +1,168 @@
+import {
+    Column,
+    Entity,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+} from "typeorm";
+import { Length } from "class-validator";
+import Messages from "../utils/messages";
+
+
+/**
+ * @namespace Entities
+ * @class Place
+ */
+@Entity()
+export class Place {
+    @PrimaryGeneratedColumn()
+    private id: number;
+
+    @Column({ length: 64 })
+    @Length(3, 64, {
+        message: Messages.validation.place_name_length,
+    })
+    private name: string;
+
+    @Column()
+    private category_id: number;
```

Comment:
This should be a relation, right?

Reply:
What if a place needs to have more than a single category?

Reply:
Done